Docs/Getting Started Guide.md

```diff
@@ -81,7 +77,7 @@ struct AppState: StateType {
 // This action does not have state and is a mere marker of "X happened":
 struct AddAction: Action { }
 
-func appReducer(action: Action, state: AppState?) -> AppState {
+func appReducer(_ action: Action, _ state: AppState?) -> AppState {
```

Comment:
Line 150 is `appReducer(action: Action, state: State?)` with named parameters, making this inconsistent. I personally prefer named parameters, especially since the function name isn't revealing what the 2 parameters would be, unlike, say `add(x, y)`.